WalletWasabi/WabiSabi/Backend/Banning/CoinVerifierApiClient.cs

```diff
@@ -53,6 +59,16 @@ public virtual async Task<ApiResponseItem> SendRequestAsync(Script script, Cance
 
 		HttpResponseMessage? response = null;
 
+		while (CurrentParallelRequestCount >= MaxParallelRequestCount)
```

Comment:
Take a look at this https://learn.microsoft.com/en-us/dotnet/api/system.threading.semaphoreslim?view=net-7.0